plugins/inventory/proxmox.py

```diff
@@ -307,6 +307,19 @@ def _get_vm_status(self, node, vmid, vmtype, name):
         status_key = self.to_safe('%s%s' % (self.get_option('facts_prefix'), status_key.lower()))
         self.inventory.set_variable(name, status_key, status)
 
+    def _get_vm_snapshots(self, node, vmid, vmtype, name):
+        ret = self._get_json("%s/api2/json/nodes/%s/%s/%s/snapshot" % (self.proxmox_url, node, vmtype, vmid))
+
+        snapshots_key = 'snapshots'
+        snapshots_key = self.to_safe('%s%s' % (self.get_option('facts_prefix'), snapshots_key.lower()))
+
+        snapshots = []
+
+        for snapshot in ret:
+            if snapshot['name'] != 'current':
+                snapshots.append(snapshot['name'])
```

Comment:
How about the following?
```suggestion
        snapshots = [snapshot['name'] for snapshot in ret if snapshot['name'] != 'current']
```

Reply:
Thanks for the tip! (I'm not a frequent programmer ;-) )